pom.xml

```diff
@@ -21,7 +21,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <javaVersion>7</javaVersion>
+    <maven.compiler.release>7</maven.compiler.release>
```

Comment:
The code below still refers to `${javaVersion}` (though that does not seem to have any effect?).

Reply:
Hmm, yes, I think we should be able to delete that.